<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>52</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>일정관리 프로그램</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="monthNavigationLayout">
      <item>
       <widget class="QPushButton" name="shift_left">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icon/icon/left-512.png</normaloff>:/icon/icon/left-512.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="monthLabel">
        <property name="text">
         <string>2025-04</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="shift_right">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icon/icon/right-512.png</normaloff>:/icon/icon/right-512.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="verticalSpacing">
       <number>0</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
